docs/reference/search/suggesters/misc.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[returning-suggesters-type]]
+=== Returning the type of the suggester
```

Comment:
Yes, I think it is still useful to document it correctly, that might be useful for others clients too.